content/en/docs/zero-code/java/spring-boot-starter/annotations.md

```diff
@@ -1,57 +1,84 @@
 doing too much code change.
 
-## Available annotations
-
-This feature uses spring-aop to wrap methods annotated with `@WithSpan` in a
-span. The arguments to the method can be captured as attributed on the created
-span by annotating the method parameters with `@SpanAttribute`.
-
-> **Note**: this annotation can only be applied to bean methods managed by the
-> spring application context. To learn more about aspect weaving in spring, see
-> [spring-aop](https://docs.spring.io/spring/docs/current/spring-framework-reference/core.html#aop).
-
-| Feature     | Property                                   | Default Value | Description                       |
-| ----------- | ------------------------------------------ | ------------- | --------------------------------- |
-| `@WithSpan` | `otel.instrumentation.annotations.enabled` | true          | Enables the WithSpan annotations. |
+If you add the `WithSpan` annotation to a method, the method is wrapped in a span. The `SpanAttribute` annotation allows you to capture the method arguments as attributes.
 
 ```java
-import org.springframework.stereotype.Component;
-
 import io.opentelemetry.instrumentation.annotations.SpanAttribute;
 import io.opentelemetry.instrumentation.annotations.WithSpan;
-import io.opentelemetry.api.trace.Span;
-import io.opentelemetry.api.trace.SpanKind;
-
-/**
- * Test WithSpan
- */
-@Component
-public class TracedClass {
 
     @WithSpan
-    public void tracedMethod() {
+    public void tracedMethod(@SpanAttribute parameter) {
     }
+```
 
-    @WithSpan(value="span name")
-    public void tracedMethodWithName() {
-        Span currentSpan = Span.current();
-        currentSpan.addEvent("ADD EVENT TO tracedMethodWithName SPAN");
-        currentSpan.setAttribute("isTestAttribute", true);
-    }
+{{% alert title="Note" color="info" %}}
+The OpenTelemetry annotations use Spring AOP based on proxys. 
+
+These annotations work only for the methods of the proxy. You can learn more in the [Spring documentation](https://docs.spring.io/spring-framework/reference/core/aop/proxying.html).
+
+In the following example, the `WithSpan` annotation won't do anything when the GET endpoint is called:
 
-    @WithSpan(kind = SpanKind.CLIENT)
-    public void tracedClientSpan() {
+```java
+@RestController
+public class MyControllerManagedBySpring {
+  
+    @GetMapping("/ping")
+    public void aMethod() {
+        anotherMethod();
     }
 
-    public void tracedMethodWithAttribute(@SpanAttribute("attributeName") String parameter) {
+    @WithSpan
+    public void anotherMethod() {
     }
 }
 ```
+
+{{% /alert %}}
+
+
+{{% alert title="Note" color="info" %}}
+
+To be able to use the OpenTelemetry annotations, you have to add the Spring Boot Starter AOP dependency to your project:
+{{< tabpane text=true >}} 
+{{% tab header="Maven (`pom.xml`)" lang=Maven %}}
+<dependencies>
+  <dependency>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-aop</artifactId>
```

Comment:
will be fixed with https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/11775